docs/pages/integrations/abnormal/abnormal.mdx

```diff
@@ -12,6 +12,12 @@
 | --------- | ------------------------------------ |
 | `api_key` | Paste the API key from Abnormal here |
 
-5. Click on the **Save** button.
-
 ![Abnormal Secrets Setup](/abnormal_secrets.png)
+
+Alternatively, you can use the following CLI command structure:
+
+```bash
+admyral secret set abnormal_secret --value api_key=your_key_value
+```
```

Comment:
Sorry, I just noticed that putting the CLI command here doesn't make sense because it mixes the UI steps with the CLI. Can we move them to the bottom after the last step for clearer separation?